openmetadata-ui/src/main/resources/ui/src/components/Lineage/Lineage.component.tsx

```diff
@@ -146,6 +153,7 @@ const Lineage = ({
               onlyRenderVisibleElements
               className="custom-react-flow"
               data-testid="react-flow-component"
+              deleteKeyCode={null}
```

Comment:
Are we blocking delete key functionality here?

Reply:
yes we are blocking default delete action. 
We are using delete and backspace action on edit mode only.